libs/langchain/langchain/retrievers/multi_vector.py

```diff
@@ -64,8 +66,17 @@ def _get_relevant_documents(
             sub_docs = self.vectorstore.max_marginal_relevance_search(
                 query, **self.search_kwargs
             )
-        else:
+        elif self.search_type == SearchType.similarity:
```

Comment:
I would suggest keeping the `else` condition as it was with the new condition above to keep the same outcome for those who rely on this method.

```python
        elif self.search_type == SearchType.similarity_score_threshold:
            sub_docs_and_similarities = (
                self.vectorstore.similarity_search_with_relevance_scores(
                    query, **self.search_kwargs
                )
            )
            sub_docs = [sub_doc for sub_doc, _ in sub_docs_and_similarities]
        else:
            sub_docs = self.vectorstore.similarity_search(query, **self.search_kwargs)
```

I suggest to apply the same changes in the `_aget_relevant_documents` method.

Reply:
ok, good suggestion! I will do it

Reply:
done